bin/hubot

```diff
@@ -3,8 +3,11 @@
 set -e
 
 fail() {
+    # Added `| systemd-cat -p warning` for el8 support
     echo "$1"
+    echo "$1" | systemd-cat -p warning
```

Comment:
That looks valid, but pretty hacky way of handling it.
What happens on systems that don't have systemd and `systemd-cat` binary avaialble?

Reply:
Fixed with @armab's idea: https://github.com/StackStorm/st2chatops/pull/140#pullrequestreview-349387498